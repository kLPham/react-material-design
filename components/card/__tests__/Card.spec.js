import React from 'react';
import { shallow } from 'enzyme';
import Card from '../index';
import CardTitle from '../cardTitle';

describe('<Card />', () => {
    it('should render', () => {
        const card = shallow(
            <Card>
                <CardTitle title="title" />
            </Card>,
    );
        expect(card).toMatchSnapshot();
    });
});
