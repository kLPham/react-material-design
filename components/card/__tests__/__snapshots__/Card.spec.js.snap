// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Card /> should render 1`] = `
<div
  className="mdc-card"
>
  <CardTitle
    larger={false}
    subtitle=""
    title="title"
  />
</div>
`;
